sdk/eventhub/event-hubs/src/loadBalancerStrategies/balancedStrategy.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { PartitionOwnership } from "../eventProcessor";
+import { LoadBalancingStrategy, identifyClaimablePartitions } from "./loadBalancingStrategy";
+
+/**
+ * The BalancedLoadBalancerStrategy is meant to be used when the user
+ * wants to reach a load balanced state with less partition 'thrashing'.
+ *
+ * Partition trashing - where a partition changes owners - is minimized
+ * by only returning a single partition to claim at a time.
+ * This minimizes the number of times a partition should need to be stolen.
+ * @internal
+ * @ignore
+ */
+export class BalancedLoadBalancingStrategy implements LoadBalancingStrategy {
+  /**
+   * Creates an instance of BalancedLoadBalancingStrategy.
+   *
+   * @param _partitionOwnershipExpirationIntervalInMs The length of time a partition claim is valid.
+   */
+  constructor(private readonly _partitionOwnershipExpirationIntervalInMs: number) {}
+
+  /**
+   * Implements load balancing by taking into account current ownership and
+   * the full set of partitions in the Event Hub.
+   * @param ourOwnerId The id we should assume is _our_ id when checking for ownership.
+   * @param claimedPartitionOwnershipMap The current claimed ownerships for partitions.
+   * @param partitionIds Partitions to assign owners to.
+   * @returns Partition ids to claim.
+   */
+  public identifyPartitionsToClaim(
+    ourOwnerId: string,
+    claimedPartitionOwnershipMap: Map<string, PartitionOwnership>,
+    partitionIds: string[]
+  ): string[] {
+    const claimablePartitions = identifyClaimablePartitions(
```

Comment:
`identifyClaimablePartitions` and `identityPartitionsToClaim` are quite similar in name.

Maybe:
* `identifyClaimablePartitions` => `listAvailablePartitions`?
* `identifyPartitionsToClaim` => `getPartitionsToClaim`?

Reply:
Thanks, I like those renames!

Reply:
Renamed!